storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/pkr-response.cpp

```diff
@@ -267,8 +268,10 @@ RS_Status PKRResponse::Append_char(const char *colName, const char *fromBuff, Ui
                            std::to_string(GetRemainingCapacity()) + std::string(" Required: ") +
                            std::to_string(estimatedBytes));
   }
+
   // from_buffer -> printable string  -> escaped string
-  char tempBuff[estimatedBytes];
+  // FIXME: Find more suitable (smaller) size
+  char tempBuff[MAX_TUPLE_SIZE_IN_WORDS * 4];
```

Comment:
buffer overflow. max data column can be 30 kb. if it is base64 encoded then it will take 41 kb. create a constant